### DISCUSSION TITLE
Need Clean Build for lineage-17.1

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#18340] [Q&A]
I have made some changes in some repositories but server is always build with the old changes and the new one isnt included, tried 2-3 times

local manifest: https://github.com/SwapnilVicky/local_manifest/tree/10

Build: https://cirrus-ci.com/build/6205155226419200

Asking for a clean build so that the new changes can take place

**Comment by Apon77** [comment]
which changes you included? 
how you got sure that its not taking new changes?
imo clean build is not necessary

**Reply by SwapnilVicky** [reply]
I have tracked some repos from fork lineage 17.1 and now its apps were included in the build, those apps should be removed now but still they make their presence, those apps were DuckDuck go browser, shuffle music player etc

**Reply by Apon77** [reply]
where you added those and which commit you remove those? give links

**Reply by SwapnilVicky** [reply]
I'm just adding these changes:-

framework/base: https://github.com/FlexLineage/android_frameworks_base/commits/lineage-17.1
vendor/lineage: https://github.com/FlexLineage/android_vendor_lineage/commits/lineage-17.1
build/make: https://github.com/FlexLineage/android_build/commits/lineage-17.1
system/core: https://github.com/FlexLineage/android_system_core/commits/lineage-17.1
packages/apps/settings: https://github.com/FlexLineage/android_packages_apps_Settings/commits/lineage-17.1

The old commits of forklineage has been removed from here

**Reply by Apon77** [reply]
where is duckduckgo, shuffle?? you are telling which you removed?

**Reply by SwapnilVicky** [reply]
i reset the commits sir, took a fresh base of lineage and picked the commits i need

**Reply by SwapnilVicky** [reply]
okay no problem BTW, thanks for your help sir ☺️
